search-commons/src/test/java/no/unit/nva/search/IndexingClientTest.java

```diff
@@ -87,6 +91,36 @@ void shouldSendIndexRequestWithIndexNameSpecifiedByIndexDocument() throws IOExce
         assertThat(extractDocumentFromSubmittedIndexRequest(), is(equalTo(indexDocument.getResource())));
     }
 
+    @Test
+    void shouldVerifyIndicesClientCreatedToAccessIndicesAPI() {
+        IndicesClientWrapper expectedIndicesClientWrapper = createMockIndicesClientWrapper();
+        when(esClient.indices()).thenReturn(expectedIndicesClientWrapper);
+        var actualIndicesClientWrapper = indexingClient.getIndicesClientWrapper();
+        assertEquals(expectedIndicesClientWrapper, actualIndicesClientWrapper);
+    }
```

Comment:
do we actually need this?

Reply:
Going to remove getIndicesClientWrapper method then this test itself has no scope